### PR TITLE
Backend/#090 order placing logic

### LINKED ISSUE #90: BACKEND | add order placing logic

### PR DESCRIPTION
**Pull request by vieraboschkova**
closes #90

### CODE REVIEW

File: api/seat/Seat.js [hunk 1]
```diff
@@ -6,8 +6,20 @@ const seatSchema = new Schema({
...
+  },
+  row: {
+    type: Number,
+    required: true,
+  },
+  column: {
+    type: Number,
+    required: true,
+  },
+  name: {
```
Comment (paula0403) [inline api/seat/Seat.js:19]: Is this necessary?I think row and column are enough

File: api/order/Order.js [hunk 2]
```diff
@@ -24,13 +24,51 @@ const orderSchema = new Schema({
...
 });
 
+orderSchema.methods.createOrdersDependencies = async function createOrdersDependencies(
+  seats,
+  screening,
+  order,
+  cb,
+) {
+  try {
+    console.log('awaiting before', screening);
```
Comment (paula0403) [inline api/order/Order.js]: remove this, please

File: api/ticket/Ticket.js [hunk 3]
```diff
@@ -6,12 +6,19 @@ const ticketSchema = new Schema({
   screening: {
     type: Schema.Types.ObjectId,
     ref: 'Screening',
+    required: true,
   },
   row: Number,
   column: Number,
```
Comment (paula0403) [inline api/ticket/Ticket.js]: row and column are in Seat, so You can remove these keys, probably.

File: api/seat/Seat.js (hunk 1 above)
Comment (vieraboschkova) [reply api/seat/Seat.js:19]: I found it useful for iterating and search purposes. I changed it to rows and columns though. The data to pass should be an array of tuples like so ```[[rowNr,columnNr],[rowNr,columnNr]]```

File: api/order/Order.js (hunk 2 above)
Comment (vieraboschkova) [reply api/order/Order.js]: Done

File: api/ticket/Ticket.js (hunk 3 above)
Comment (vieraboschkova) [reply api/ticket/Ticket.js]: Done

**Review by paula0403 — DISMISSED** [review]

**Review by paula0403 — APPROVED** [review]

**Pull request merged**